web/src/pages/Route/components/DataLoader/Import.tsx

```diff
@@ -103,18 +103,20 @@ const DataLoaderImport: React.FC<Props> = (props) => {
     });
     formData.append('file', uploadFileList[0]);
 
-    importRoutes(formData).then((r) => {
-      let errorNumber = 0;
-      entityNames.forEach((v) => {
-        errorNumber += r.data[v].failed;
-      });
+    importRoutes(formData)
+      .then((r) => {
+        let errorNumber = 0;
+        entityNames.forEach((v) => {
+          errorNumber += r.data[v].failed;
+        });
 
-      setImportResult({
-        success: errorNumber <= 0,
-        data: r.data,
-      });
-      setState('result');
-    });
+        setImportResult({
+          success: errorNumber <= 0,
+          data: r.data,
+        });
+        setState('result');
+      })
+      .catch(console.error);
```

Comment:
should we disable `console.error` in production environment?

Reply:
That probably means we don't need to catch and just keep an empty catch function.

Reply:
We don't actually need to use it to handle errors either, just to prevent JS errors where exceptions are not caught.

Reply:
> That probably means we don't need to catch and just keep an empty catch function. 🤔

I think `console.error` can somehow be removed at compile time for the production environment

Reply:
Maybe we can try https://stackoverflow.com/a/41041580 (Not Now)